src/promptflow/promptflow/_cli/_completers/_param_completers.py

```diff
@@ -0,0 +1,19 @@
+from argcomplete import warn
+
+from promptflow._sdk._constants import ListViewType
+from promptflow.client import PFClient
+
+
+def run_name_completer(prefix, parsed_args, **kwargs):
+    client = PFClient()
+    runs = client.runs.list(max_results=50, list_view_type=ListViewType.ALL)
```

Comment:
maybe add a constants for the magic number 50. will 50 be too much? 
I am not sure how will this looks if we have 50 results in the terminal?